### ISSUE TITLE
Jimp constructor overloading -- "no matching constructor"

### ISSUE THREAD

**Issue by zaneclaes** [Steve-Mcl/node-red-contrib-image-tools#18]
I just installed the node and am attempting to resize an image. In my logs I see:

```
29 Jul 11:36:37 - [error] [jimp-image:resize, base64] Error: No matching constructor overloading was found. Please see the docs for how to call the Jimp constructor.
```

My node (have also tried the last option left blank):
![Screen Shot 2020-07-29 at 11 41 13 AM](https://user-images.githubusercontent.com/2084821/88834012-6ea4a080-d190-11ea-8152-5ad00c616e46.png)

**Comment by Steve-Mcl** [comment]
Can you export your flow and paste it in a reply

` ``` `
` paste code like this between backticks `
` ``` `

I will take a look. I updated the node today and night have broken something

Cheers.

**Comment by Steve-Mcl** [comment]
Never mind. I spotted your error. 

Look at the type set for X & y properties. 

You need to either pass the size in or set the type to number (you have them set to `msg.` so the node is expecting you to send the X&Y values in via a msg property)

**Comment by zaneclaes** [comment]
Yeah, I just caught that mistake as well w.r.t. the fact that I did not change the default `msg.` prefix.

However, this error happens far upstream of that particular problem. In fact, it shows up in the Node Red startup logs, before the flows are even fully loaded. I think it may be related to the jimp dependency not being correctly installed initially for some reason. I ended up manually adding `"jimp": "0.2.x"` to package.json, did a `npm install`, restarted the deployment, and it started working.

**Comment by Steve-Mcl** [comment]
> "jimp": "0.2.x" is very very old.  

"jimp" is included in this package as "^0.14.0"  - how did you install this node?

what version of NODEJS are you running?
what version of node-red are you running?

**Comment by zaneclaes** [comment]
Good point about the version; should have checked that.

Node Red v1.1.2, deployed via the docker image (nodered/node-red:latest)
onto a Raspberry Pi k8s cluster with NFS persistence.

Packages are generally installed via the Node Red settings/palette, though
I do occasionally SSH into the pod and edit /data/package.json to do a
manual npm install (as I did to add the dependency, here). I find that I
sometimes have permissions problems with the docker container, so that may
be at play here. Unfortunately, the pod which did the original install was
already deleted, and I only have my o11y logs set to ~48h retention.
